### PR TITLE
Generate and categorize changelog items for RG 2.7.0 releases.

### PR DESCRIPTION
**Pull request by hsbt**
@segiddins I create and categorized pull requests used by `util/update_changelog.rb`.

/cc @tenderlove

### CODE REVIEW

**Comment by hsbt** [comment]
@bundlerbot r+

**Comment by segiddins** [comment]
Didn’t have a chance to review before you merged, but I was hoping to get Bundler 1.16 into this (I’m trying to release 1.16 now)

**Comment by hsbt** [comment]
I see. I will merge and add an entry for bundler-1.16.0 release.

If you have other item for RG 2.7.0 release, Please add it into History.txt.

File: History.txt [hunk 1]
```diff
@@ -1,5 +1,168 @@
...
+* [TestCase] Address comments around ui changes. Pull request #1677 by
+  Samuel Giddins.
+* Eagerly resolve in activate_bin_path. Pull request #1666 by Samuel
+  Giddins.
+* [Version] Make hash based upon canonical segments. Pull request #1659 by
+  Samuel Giddins.
+
+Compatibility changes:
+
+* ubygems.rb is unavailable on Ruby 2.5. Pull request #2028 #2027 #2029
```
Comment (segiddins) [inline History.txt]: that's not true, we haven't removed `ubygems.rb` from this repo

**Comment by hsbt** [comment]
@bundlerbot retry

**Comment by hsbt** [comment]
@bundlerbot r+

**Pull request merged**